test/await-after.test.js

```diff
@@ -4,6 +4,8 @@ const { test } = require('tap')
 const boot = require('..')
 const { promisify } = require('util')
 const sleep = promisify(setTimeout)
+const fs = require('fs').promises
```

Comment:
The most recent CI failure was due to an FS promises import that didn't work with Node 12.  This updated import should work w/ Node 12/14/16